modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -538,6 +538,12 @@ def update_appointment_vaos(appt_id, status)
         params = VAOS::V2::UpdateAppointmentForm.new(status:).params
         perform(:put, url_path, params, headers)
       end
+
+      def validate_response_schema(response, contract_name)
+        return unless response.success? && response.body[:data].present?
```

Comment:
Adding this guard because only 2 of the first 25 runs had actual data in them. 17 were just empty and 6 had upstream failures (so the data was also empty).